crashreports/BreakpadConfigurer.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <string>
+#include <map>
+
+class BreakpadConfigurer final
```

Comment:
final is good.

The class needs a few comments as to purpose, and make the habit to use Doxygen commands too.  At least:

`
//! A short description
`
or
`/*!
A long description
*/
`
before this line.

I had to read AudacityApp.cpp to understand its purpose:  use it at startup time to initialize crash reporting.  Also that this object needs only a short lifetime, unlike the CrashReportContext